### PR TITLE
Prototype build

### PR DESCRIPTION
**Pull request by JohnByrneJames**
Changes

### CODE REVIEW

**Review by Rokkii — DISMISSED** [review]
Profanity aside xD  LGTM

File: UnityClient/Assets/Scripts/AStar/Graph.cs [hunk 1]
```diff
@@ -201,6 +202,12 @@ public Node NodeFromWorldPoint(Vector3 worldPosition)
     {
         float percentX = (worldPosition.x + gridWorldSize.x / 2) / gridWorldSize.x;
         float percentY = (worldPosition.z + gridWorldSize.y / 2) / gridWorldSize.y;
+
+/*        if (percentX > 1 || percentX < 0 || percentY > 1 || percentY < 0)
```
Comment (RubyNova) [inline UnityClient/Assets/Scripts/AStar/Graph.cs]: This is dead code, should delete this probably. Lol.

File: UnityClient/Assets/Scripts/AStar/Pathfinding.cs [hunk 2]
```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using System;
 using UnityEngine;
+using Debug = UnityEngine.Debug;
```
Comment (RubyNova) [inline UnityClient/Assets/Scripts/AStar/Pathfinding.cs:7]: Why the alias?

File: UnityClient/Assets/Scripts/AStar/Pathfinding.cs [hunk 3]
```diff
@@ -24,6 +25,7 @@ public void StartFindPath(Vector3 startPos, Vector3 targetPos)
 
     IEnumerator FindPath(Vector3 startPos, Vector3 targetPos)
```
Comment (RubyNova) [inline UnityClient/Assets/Scripts/AStar/Pathfinding.cs]: Accessor is missing. Please provide an accessor.

**Review by RubyNova — CHANGES_REQUESTED** [review]

**Review by RubyNova — APPROVED** [review]
verly loosely LGTM. Further refactoring should be propagated to another PR.

**Pull request merged**